File: algorithms/code/leetcode/lc013_roman_to_integer/LC013RomanToInteger.Tests.cs
using Xunit;

namespace AlgorithmDesign.code.leetcode.lc013_roman_to_integer
{
    public class SolutionTests
    {
        [Fact]
        public void TestMcmliv()
        {
            Assert.Equal(1954, new Solution().RomanToInt("MCMLIV"));
        }

        [Fact]
        public void TestNothing()
        {
            Assert.Equal(0, new Solution().RomanToInt(null));
        }
    }
}

File: algorithms/code/leetcode/lc013_roman_to_integer/LC013RomanToIntegerTests.java
package leetcode.lc013_roman_to_integer;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public final class LC013RomanToIntegerTests {
    @Test
    public void testMCMLIV() throws Exception {
        LC013RomanToInteger solution;
        solution = new LC013RomanToInteger();
        assertEquals(1954, solution.romanToInt("MCMLIV"));
    }

    @Test
    public void testNothing() throws Exception {
        LC013RomanToInteger solution;
        solution = new LC013RomanToInteger();
        assertEquals(0, solution.romanToInt(null));
    }
}
